docs/source/release/v3.8.0/framework.rst

```diff
@@ -13,7 +13,9 @@ Framework Changes
   is almost identical, but a small number of cases (such as adding the workspaces ``Z`` and ``z``) will work
   in a more predictable manner.
 
-- A race condition when accessing a singleton from multiple threads was fixed. 
+- A race condition when accessing a singleton from multiple threads was fixed.
+
+- Log file buffers are no longer flushed by default for each newline received, increasing the speed of some system tests on Windows by 3x. 
```

Comment:
Turning of flushing to disk makes `HFIRTestsAPIv2.HFIRTestsAPIv2` run even faster than with just commenting out `ThreadSafeLogStreamBuf::writeToDevice(...)`.  In that case, the speedup between master and this pull request is closer to 4.5x